lib/pipeline.js

```diff
@@ -471,22 +471,10 @@ class PipelineModel extends BaseModel {
      * @property admin
      * @return Promise
     */
-    // TODO: Get the first admin. Validate if it's valid. If not, remove from admins field,
-    //       and retry with the next admin.
     get admin() {
         delete this.admin;
 
-        // Lazy load factory dependency to prevent circular dependency issues
-        // https://nodejs.org/api/modules.html#modules_cycles
-        /* eslint-disable global-require */
-        const UserFactory = require('./userFactory');
-        /* eslint-enable global-require */
-        const factory = UserFactory.getInstance();
-
-        const admin = factory.get({
-            username: Object.keys(this.admins)[0],
-            scmContext: this.scmContext
-        });
+        const admin = this.getFirstAdmin();
```

Comment:
Why not just `return this.getFirstAdmin()`?

Reply:
It is because `admin` is used [here](https://github.com/sonic-screwdriver-cd/models/blob/1e4da9ca7e82ec42f747499ecb488430c778a551/lib/pipeline.js#L484). I think it is used `admin` as property.

Reply:
@minz1027 
> since this.getFirstAdmin() is an async function, do we need await here?

Only `async` function can have `await`, and `getter` seems not to be able to have `async`.

It might not need to add await in `this.GetFirstAdmin()`.

If `pipepline.admin` is called from `async` function, I await like [here](https://github.com/screwdriver-cd/executor-queue/pull/25/files#diff-168726dbe96b3ce427e7fedce31bb0bcR166). If `pipeline.admin` is in NOT `async` function, `admin` is referred in `then` context like [here](https://github.com/screwdriver-cd/screwdriver/pull/1061/files#diff-7a514e440aa937beee722c7ac84dfb93R224).